Ruby-Project/student_class.rb

```diff
@@ -8,4 +10,9 @@ def initialize(age, classroom, name = 'Unknown', parent_permission: true)
   def play_hooky
     '¯(ツ)/¯'
   end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.Student.push(self) unless classroom.Student.include?(self)
```

Comment:
Kindly replace `Student` with `student` considering you used a lowercase `s` when naming `@student` in your `Classroom` class.
```
classroom.student.push(self) unless classroom.student.include?(self)
```